java/.ci/UiJenkinsfile

```diff
@@ -0,0 +1,468 @@
+                                    --region="us-west1" \
+                                    --version="1.1" \
+                                    --jars="${jarPath},gs://dataproc-templates/jars/mysql-connector-java.jar" \
+                                    --subnet="projects/yadavaja-sandbox/regions/us-west1/subnetworks/test-subnet1" \
+                                    -- --template=JDBCTOGCS \
+                                    --templateProperty project.id="${GCP_PROJECT}" \
+                                    --templateProperty jdbctogcs.jdbc.url="${TEST_JDBC_URL}" \
+                                    --templateProperty jdbctogcs.jdbc.driver.class.name="com.mysql.jdbc.Driver" \
+                                    --templateProperty jdbctogcs.output.format="csv" \
+                                    --templateProperty jdbctogcs.output.location="gs://dataproc-templates/integration-testing/output/JDBCTOGCS/csv" \
+                                    --templateProperty jdbctogcs.sql="select * from test.employee" \
+                                    --templateProperty jdbctogcs.write.mode="overwrite" 
+                            '''
+                        }
+                    }
+                }
+                stage('JDBC TO GCS (avro)'){
+                    steps {
+                        retry(count: stageRetryCount) {
+                            sh '''
+                                gcloud dataproc batches submit spark \
+                                    --class=com.google.cloud.dataproc.templates.main.DataProcTemplate \
+                                    --project="${GCP_PROJECT}" \
+                                    --region="us-west1" \
+                                    --version="1.1" \
+                                    --jars="${jarPath},gs://dataproc-templates/jars/mysql-connector-java.jar,file:///usr/lib/spark/external/spark-avro.jar" \
+                                    --subnet="projects/yadavaja-sandbox/regions/us-west1/subnetworks/test-subnet1" \
+                                    -- --template=JDBCTOGCS \
+                                    --templateProperty project.id="${GCP_PROJECT}" \
+                                    --templateProperty jdbctogcs.jdbc.url="${TEST_JDBC_URL}" \
+                                    --templateProperty jdbctogcs.jdbc.driver.class.name="com.mysql.jdbc.Driver" \
+                                    --templateProperty jdbctogcs.output.format="avro" \
+                                    --templateProperty jdbctogcs.output.location="gs://dataproc-templates/integration-testing/output/JDBCTOGCS/avro" \
+                                    --templateProperty jdbctogcs.sql="select * from test.employee" \
+                                    --templateProperty jdbctogcs.write.mode="overwrite"
+                            '''
+                        }
+                    }
+                }
+            }
+        }
+    }
+    // post {
```

Comment:
We can uncomment it and add a conditinal statement similar to https://github.com/GoogleCloudPlatform/dataproc-templates/blob/28c155fbf1e3322de9e9fbe376b1b5078c565042/java/.ci/Jenkinsfile#L1099

Which would trigger notifications only when it runs for main branch.